android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -703,12 +713,12 @@ public void switchCamera() {
         }
     }
 
-    public void toggleVideo(boolean enabled) {
+    public void toggleVideo(boolean enabled, String videoTrackName) {
         isVideoEnabled = enabled;
 
         if (cameraCapturer == null && enabled) {
             String fallbackCameraType = cameraType == null ? CustomTwilioVideoView.FRONT_CAMERA_TYPE : cameraType;
-            boolean createVideoStatus = createLocalVideo(true, fallbackCameraType);
+            boolean createVideoStatus = createLocalVideo(true, fallbackCameraType, videoTrackName);
```

Comment:
At the native level, we handle the `undefined` `videoTrackName` in `createLocalVideo` - while exposing the `customTrackName` on the JS bridge and interface the optional parameter.